### PR TITLE
Updated ReadMe

### PR DESCRIPTION
**Pull request by drlukeangel**
Added windows commands

### CODE REVIEW

**Comment by Hartmarken348** [comment]
Readme

File: content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md [hunk 1]
```diff
@@ -243,23 +243,30 @@ The preceding configuration file defines a new context named `dev-ramp-up`.
 
 See whether you have an environment variable named `KUBECONFIG`. If so, save the
 current value of your `KUBECONFIG` environment variable, so you can restore it later.
-For example, on Linux:
-
+For example:
```
Comment (shavidissa) [inline content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md]: Add a <br> tag here too :)

File: content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md [hunk 2]
```diff
@@ -243,23 +243,30 @@ The preceding configuration file defines a new context named `dev-ramp-up`.
...
+Temporarily append two paths to your `KUBECONFIG` environment variable. For example:<br>
 
+Linux:
 ```shell
 export  KUBECONFIG=$KUBECONFIG:config-demo:config-demo-2
 ```
-
+Windows PowerShell
+```shell
+$Env:KUBECONFIG_SAVED=(config-demo;config-demo-2)
```
Comment (kbhawkey) [inline content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md]: - Why do you need the raw line break?
- You could add two sub-headers, such as:
   ### Linux, using bash:
  ```shell
  some linux commands ...
  ```
  ### Windows Powershell:
  ```shell
  some windows commands ...
  ```
- Is this environment variable KUBECONFIG_SAVED correct?

**Comment by Rajakavitha1** [comment]
@drlukeangel could you please address the suggestions in the review comments.

**Pull request merged**